src/lib/Libattr/master_sched_attr_def.xml

```diff
@@ -386,7 +386,7 @@
 	<member_at_comp>comp_str</member_at_comp>
 	<member_at_free>free_str</member_at_free>
 	<member_at_action>NULL_FUNC</member_at_action>
-	<member_at_flags><both>READ_ONLY</both></member_at_flags>
+	<member_at_flags><both>NO_USER_SET | ATR_DFLAG_NOSAVM</both></member_at_flags>
```

Comment:
@arungrover 
Till we implement the next interface where sever itself controls scheduler i.e. starting, stopping etc we need to allow the setting of this attribute by operator and manager. The reason is given below.

Let us say if admin has configured 5 schedulers but he wants to use only two schedulers at the moment. In this case if "state" is read-only then server keeps on contacting each scheduler i.e. keep calling contact_sched() function infinitely until it succeeds which is as per the existing design only. So to avoid this I have made "state" as writable.  I also noticed that this is in line with the "state" attribute of the node. Thought of updating the EDD only after reviewers also like this idea.

Reply:
@suresh-thelkar I thought the point of the scheduling attribute was to control this.  Why would the server keep trying to contact a scheduler if the scheduling attribute was set to false?

Reply:
Oh, and the node state is not like the scheduler state.  There are node states that the admin sets (e.g., offline).  There is no such scheduling state.  PBS is in complete control of the scheduling state.

Reply:
@bhroam 
Server does not contact the corresponding schedulers if scheduling is false.
As you suggested, I will use the scheduling attribute itself to control this.

Reply:
Also I have done code changes accordingly.